src/network/connection.cpp

```diff
@@ -241,28 +236,20 @@ BufferedPacket ReliablePacketBuffer::popSeqnum(u16 seqnum)
 				<< " not found in reliable buffer"<<std::endl);
 		throw NotFoundException("seqnum not found in buffer");
 	}
-	BufferedPacket p = *r;
-
-
-	RPBSearchResult next = r;
```

Comment:
Why is this part no longer needed?

Reply:
`m_oldest_non_answered_ack` is always written at the end of the function so this was just dead code.